src/test/java/com/mindee/extraction/PDFExtractorTest.java

```diff
@@ -0,0 +1,66 @@
+package com.mindee.extraction;
+
+import com.fasterxml.jackson.databind.JavaType;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.mindee.input.LocalInputSource;
+import com.mindee.parsing.common.PredictResponse;
+import com.mindee.product.invoicesplitter.InvoiceSplitterV1;
+import java.io.File;
+import java.io.IOException;
+import java.util.List;
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.Test;
+
+public class PDFExtractorTest {
+
+  protected PredictResponse<InvoiceSplitterV1> getInvoiceSplitterPrediction() throws
+    IOException {
+    ObjectMapper objectMapper = new ObjectMapper();
+    objectMapper.findAndRegisterModules();
+
+    JavaType type = objectMapper.getTypeFactory().constructParametricType(
+      PredictResponse.class,
+      InvoiceSplitterV1.class
+    );
+    return objectMapper.readValue(
+      new File("src/test/resources/products/invoice_splitter/response_v1/" + "complete" + ".json"),
```

Comment:
```suggestion
      new File("src/test/resources/products/invoice_splitter/response_v1/complete.json"),
```

Reply:
Yeah the linter still isn't fully following maven's settings, idk how to fix it :(